script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -136,6 +136,7 @@ Everything up to Rank III are 'living corpses'.
 -- Ⅳ～Ⅴ階梯目は夜を生きる『夜属』、
 // 夜を生きる translated here as nocturnal, since otherwise it's like
 // "night dwellers who live at night" which is ??? eigo
+// Mark up on night dwellers
```

Comment:
this is one for lore folks. @yk1512 you have the floor

Reply:
I would say use 'nightkin', keeping nocturnal should be fine.